code/modules/mob/living/carbon/diona_base.dm

```diff
@@ -0,0 +1,548 @@
+//This function is for code that is shared by diona nymphs and gestalt
+/mob/living/carbon/var/stored_energy
+/mob/living/carbon/var/max_energy = 90
+
+
+
+#define FLASHLIGHT_STRENGTH	1.59375
```

Comment:
Please look over these, and make sure they're #undef-d properly if they're only used in this file.